app/pages/lab/workflow.cjsx

```diff
@@ -53,6 +54,11 @@ EditWorkflowPage = React.createClass
     deletionError: null
     workflowCreationInProgress: false
 
+  componentWillReceiveProps: (nextProps) ->
+    if nextProps.workflow isnt @props.workflow
+      ReactDOM.findDOMNode(@refs.defaultWorkflow).defaultChecked = nextProps.project.configuration.default_workflow is nextProps.workflow.id
```

Comment:
OH I get it now! Sorry! I think there's a neater way to handle that, let me play with it a minute...